flowauth/backend/flowauth/main.py

```diff
@@ -85,20 +85,18 @@ def handle_invalid_usage(error):
     return response
 
 
-def handle_integrity_error(error):
+def handle_unique_error(error):
     """Returns violations of UNIQUE constraints specifically, otherwise reraise"""
-    from flowauth.models import db
-
     print(error)
-    _, _, error_message = error.args[0].partition(" ")
+    _, _, error_message = error.args[-1].partition(" ")
```

Comment:
That line looks a bit cryptic to me and the `[-1]` to me is asking for an exception but I'll say I trust you know what you're doing.

Reply:
Point - sorted.